### PR TITLE
updated get_spacings() to calculate dxy from 1D or 2D lat-lon values, independent of their dimension order 

### PR DESCRIPTION
**Pull request by JuliaKukulies**
I updated `get_spacings()` in `utils/general.py` because previously this function assumed that for datasets with lon-lat coordinates, these were two-dimensional with 'lon' being the first dimension and 'lat' being the second dimension. I mistakenly introduced this with #65, and I think it is better to make the function more general so that it works for both 1D and 2D lat and lons and independent of their dimension order.

### CODE REVIEW

**Review by w-k-jones — APPROVED** [review]

File: tobac/utils/general.py
Comment (freemansw1) [inline tobac/utils/general.py:270]: Should we just loop through all of the coordinates and find `latitude` and `longitude` that way?

Comment (JuliaKukulies) [reply tobac/utils/general.py]: Yes, good suggestion! I changed this part in the code.

**Review by freemansw1 — APPROVED** [review]

**Comment by freemansw1** [comment]
What is our policy on merging? Should @JuliaKukulies merge this, or should I given that I was the last reviewer?

**Comment by w-k-jones** [comment]
We should probably set a policy for this. Personally I think it's best to leave it to whoever made the pull request in case they need to make any further additions/changes

**Comment by freemansw1** [comment]
Agreed with that, for PRs done by maintainers. For non-maintainers, maybe after the last review, ask (and tag them) about whether they are ready to merge?

I'll leave it to @JuliaKukulies for this one.

**Comment by JuliaKukulies** [comment]
Sounds good, I agree with your suggestion @freemansw1  to leave it to the last reviewer for non-maintainers, but communicate with the person if they are ready. Good that you brought that up! Sorry @w-k-jones for having merged your last PR on the notebook update- next time I will leave it to you

**Comment by w-k-jones** [comment]
> Sorry @w-k-jones for having merged your last PR on the notebook update- next time I will leave it to you

No worries! It's not something we have a specific policy on yet, and often it's definitely easier for the approving reviewer to merge

**Pull request merged**
